src/main/java/Storage.java

```diff
@@ -0,0 +1,65 @@
+import java.io.*;
```

Comment:
You can consider listing explicitly the classes that you are importing!

Reply:
Okay!